.github/workflows/php.yml

```diff
@@ -0,0 +1,148 @@
+---
+name: PHP
+
+on:
+  # See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+
+  # See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push
+  push:
+    branches:
+      - '*'         # matches every branch that doesn't contain a '/'
+      - '*/*'       # matches every branch containing a single '/'
+      - '**'        # matches every branch
+    tags:
+      - "*"
+
+  # See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+env:
+  EXTRA_PHP_EXTENSIONS: intl bcmath zip pcntl exif curl gd pdo_pgsql pdo_mysql pdo_sqlite
+  DOCKER_PHP_EXTENSION_INSTALLER_VERSION: 2.2.2
+  COMPOSER_VERSION: 2.7.1
+
+jobs:
+  test:
```

Comment:
I'd probably recommend splitting this into a lint and test jobs, such that you can fail the workflow fast on linting issues? Though you are using `fail-fast` false below?

Reply:
I merged the two into for a couple of reasons:

1. Avoiding confusing "cause and effect" situations where a syntax error caused many pipelines to fail (and having to figure out which) - or tests failing under PHP 8.x fail but not on PHP 8.y due to some syntax/issue.
2. There is no point in running tests (or its pipeline) if there is a syntax error, so it shortens full pipeline runtime from end-to-end.
3. Fewer CI jobs kicking off at the same time, to keep the number of CI jobs ~low. 

I'm going to pull out the `pint` part for now though, as @dansup seem to want to do this a different way / timing.